homeassistant/components/nasweb/config_flow.py

```diff
@@ -102,6 +102,8 @@ async def async_step_user(
                 _LOGGER.exception("Unexpected exception")
                 errors["base"] = "unknown"
             else:
+                await self.async_set_unique_id(info[CONF_UNIQUE_ID])
+                self._abort_if_unique_id_configured({CONF_HOST: user_input[CONF_HOST]})
```

Comment:
Why do we update the host of the existing entry here?

Reply:
It was my lack of understanding, now I see that it should be called here without any arguments